src/platform/utilities/tests/feature-toggles/helpers.unit.spec.js

```diff
@@ -0,0 +1,68 @@
+import { expect } from 'chai';
+
+const {
+  setCookie,
+  createTokenFromCookie,
+  getFlipperId,
+} = require('../../feature-toggles/helpers');
+
+describe('feature toogles', () => {
+  describe('setCookie', () => {
+    it('should return the cookie if it already exists', () => {
+      const token = 't68mzm0skza1zqwinl4byt';
+      document.cookie = `newcookieName=${token}`;
+      expect(setCookie('newcookieName', 'differentToken')).to.deep.equal(
+        document.cookie.replace(
+          // eslint-disable-next-line no-useless-escape
+          /(?:(?:^|.*;\s*)newcookieName\s*\=\s*([^;]*).*$)|^.*$/,
+          '$1',
+        ),
+      );
+    });
+
+    it('should set a new cookie if one does not already exist', () => {
+      const token = 't68mzm0skza1zqwinl4byt';
+      expect(setCookie('cookieName', token)).to.deep.equal(token);
+    });
+  });
+
+  describe('createTokenFromCookie', () => {
+    it('should return the cookie if it already exists', () => {
+      const token = 'g84k9d0skza1zqwinl4byt';
+      document.cookie = `newcookieName=${token}`;
+      expect(createTokenFromCookie('newcookieName')).to.deep.equal(
+        document.cookie.replace(
+          // eslint-disable-next-line no-useless-escape
+          /(?:(?:^|.*;\s*)newcookieName\s*\=\s*([^;]*).*$)|^.*$/,
+          '$1',
+        ),
+      );
+    });
+
+    it('should return a random string if the cookie does not already exist', () => {
+      expect(createTokenFromCookie('newcookieName')).to.not.deep.equal(
```

Comment:
Make sure you set `document.cookie = ''` before this; otherwise the test will be unpredictable. (If one of the other tests that does set `document.cookie` runs first, this test won't have the proper beginning state and the result will be invalid.